src/vs/workbench/contrib/tasks/browser/task.contribution.ts

```diff
@@ -495,15 +495,9 @@ configurationRegistry.registerConfiguration({
 			default: false
 		},
 		[TaskSettingId.AllowAutomaticTasks]: {
-			type: 'string',
-			enum: ['on', 'auto', 'off'],
-			enumDescriptions: [
-				nls.localize('task.allowAutomaticTasks.on', "Always"),
-				nls.localize('task.allowAutomaticTasks.auto', "Prompt for permission for each folder"),
-				nls.localize('task.allowAutomaticTasks.off', "Never"),
-			],
-			description: nls.localize('task.allowAutomaticTasks', "Enable automatic tasks in the folder - note that tasks won't run in an untrusted workspace."),
-			default: 'auto',
+			type: 'boolean',
+			description: nls.localize('task.allowAutomaticTasks', "Enable automatic tasks in any trusted workspace"),
+			default: true,
```

Comment:
I thought we'd want to still keep auto, just change the default to on?

Reply:
Oh my thought was to get rid of the prompt entirely

Reply:
The poll we did was fairly inconclusive. I would still use off, but I suspect there are those who would still want it to be a per-workspace decision. Removing the previous default outright is quite risky

Reply:
We should also consider `scope: ConfigurationScope.RESOURCE` if we want it to be team choice vs user choice, and how that would interact with auto